src/NavData.jl

```diff
@@ -70,6 +70,19 @@ function additionalNavInput(
 
     if !isnothing(acqMap) && !isnothing(acqData)
         (freq_enc_FoV, freq_enc_samples, phase_enc_FoV, phase_enc_samples) = Find_scaling_sensit(acqMap, acqData)
+        expand_freq = 0
+        expand_phase = 0
+        if freq_enc_FoV[1] < freq_enc_FoV[2]
+            @warn "The reference data field of view is smaller than the image data field of view."
+            expand_freq = ceil(Int64, (freq_enc_FoV[2] - freq_enc_FoV[1]) / (freq_enc_FoV[1]/freq_enc_samples[1]))
+        elseif phase_enc_FoV[1] < phase_enc_FoV[2]
```

Comment:
Same here, should add test cases. If it does work for the ISMRM purpose, I would say this can be added as a separate PR and handled afterwards